src/jit/lower.cpp

```diff
@@ -897,6 +969,16 @@ void Lowering::LowerNode(GenTreePtr* ppTree, Compiler::fgWalkData* data)
         LowerAdd(ppTree, data);
         break;
         
+    case GT_UDIV:
+    case GT_UMOD:
+        LowerUnsignedDivOrMod(*ppTree);
+        break;
+
+    case GT_DIV:
+    case GT_MOD:
+        LowerSignedDivOrMod(ppTree, data);
```

Comment:
I asked myself the same question but I couldn't figure out what's the established practice around using minopts so I just did what the original code did - ignored minopts. 

I'd say that minopts should be used here. Anyone else?